.github/workflows/dependabot.yml

```
@@ -0,0 +1,12 @@
+version: 2
+
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      day: "monday"
+      time: "14:00"
+      timezone: "Asia/Tokyo"
+    reviewers:
+      - "increments/shared-dev-team"
```

nit: "reviewers" is need not be specified.

```suggestion
```

---

If you must define "reviewers", `@increments/developers` is recommended.

```suggestion
    reviewers:
      - "increments/developers"
```

---

@mziyut 
Set reviewers to @increments/developers.
The reason is so that `@increments/developers` can recognize when a Dependabot is created.